res/input-leap.desktop

```diff
@@ -3,7 +3,8 @@ Type=Application
 Name=Input Leap
 Comment=Keyboard and mouse sharing solution
 Exec=input-leap
-Icon=input-leap
+Icon=input-leap.svg
```

Comment:
The icon name without svg should work correctly when inputleap is installed.

Reply:
It should, but I wanted to see if adding the extension made a difference.

Reply:
Reverted.